src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -696,34 +725,46 @@ export class PreferenceStore {
         return this.preferences.get(PreferenceKeys.LATEST_RELEASE) ?? DEFAULTS.SILENT.latestRelease;
     }
 
+    /**
+     * Sets the preference parameter
+     *
+     * @param key - The enum of {@link PreferenceKeys}.
+     * @param value - The given value to the preference key.
+     * @returns false if the key or value is not valid; yield a result using {@link ApiService.Instance.setPreference}
+     * @remarks {@link PreferenceKeys.LOG_EVENT} should be a number list containing {@link CARTA.EventType} and {@link PreferenceKeys.GLOBAL_AUTO_WCS_MATCHING} should be a bitwise-OR value of a combination of enum {@link WCSMatchingType}.
```

Comment:
I think we should instead mention that the parameter `value` actually requires enums but not a number list or a bitwise-OR value.